cfy_manager/main.py

```diff
@@ -543,45 +547,54 @@ def _print_finish_message(config_file=None):
 def _are_components_installed():
     return all(
         os.path.isfile(os.path.join(INITIAL_INSTALL_DIR, service_name))
-        for service_name in get_installed_services_names())
+        for service_name in config[SERVICES_TO_INSTALL])
```

Comment:
Shouldn't we use the installed service names here?

Reply:
We are using them, or I don't fully understand what you mean?

Reply:
Here we're using the service_to_install from the config, not the actually installed services?

Reply:
Right, so the purpose of this function is to return True if the services listed in the config.yaml are already installed (i.e. their service-name file exists). The function that @tehasdf wrote that gets the already installed srvices is this one https://github.com/cloudify-cosmo/cloudify-manager-install/blob/d1e0d8bd8ea6ce037ddd7fc4390b40ca405977ab/cfy_manager/config.py#L61

Reply:
@YoniItzhak I have the same concerns raised by sandy about using the services from the config vs the services which already installed ? should not we really use/reference the installed services not the service from the config ?

Reply:
@mabuaisha notice that the function does check whether a file for each service name in the /etc/cloudify/.installed folder

Reply:
@glukhman yeah but the check is parse the `config[SERVICES_TO_INSTALL])` not the installed services

Reply:
@mabuaisha this function, `_are_components_installed` should return True if the installation of Cloudify Manager succeeded. To check this, it goes over the services that were supposed to be installed according to the config.yaml (under `services_to_install`). `get_installed_services_names()` is a function I deleted (https://github.com/cloudify-cosmo/cloudify-manager-install/pull/940/files#diff-978e942d5c92a65a424a164f1011b7dfL263-L267) and just replaced it with `config[SERVICES_TO_INSTALL]` so it would go over all the services in the `services_to_install` section and not just the main ones.